src/main/java/org/vaadin/application/views/InvoiceView.java

```diff
@@ -0,0 +1,164 @@
+package org.vaadin.application.views;
+
+import com.vaadin.flow.component.button.Button;
+import com.vaadin.flow.component.combobox.ComboBox;
+import com.vaadin.flow.component.datepicker.DatePicker;
+import com.vaadin.flow.component.formlayout.FormLayout;
+import com.vaadin.flow.component.grid.Grid;
+import com.vaadin.flow.component.html.H1;
+import com.vaadin.flow.component.notification.Notification;
+import com.vaadin.flow.component.orderedlayout.VerticalLayout;
+import com.vaadin.flow.component.textfield.NumberField;
+import com.vaadin.flow.component.textfield.TextArea;
+import com.vaadin.flow.component.textfield.TextField;
+import com.vaadin.flow.router.Route;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.vaadin.application.MainLayout;
+import org.vaadin.application.model.Invoice;
+import org.vaadin.application.service.InvoiceService;
+import org.vaadin.application.service.SessionService;
+import org.vaadin.application.service.UserService;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.UUID;
+
+@Route(value = "invoice", layout = MainLayout.class)
```

Comment:
To keep the commenting consistent across all the classes, can you please add some Javadoc comments to this class? :)

Reply:
Javadoc commenting has been added, thanks for the suggestion!